### PR TITLE
6 feature6 myflightsitem

### LINKED ISSUE #6: feature#6-MyFlightsItem
# REQUIREMENTS
As a user, I want to have access to the flight schedule list

# DESIGN
imagen

### PR DESCRIPTION
**Pull request by TonyHdez-Pi**
# Description
Implement flight item to later integrate it with the flight screen.
 
  ## Screenshots 
imagen

### CODE REVIEW

File: flight-reservation/components/FlightItemComponent/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      </View>
+      <View style={styles.date}>
+        <Text style={styles.dateText}>September 3, 2020</Text>
+        <Text style={styles.dateText}>2 Passengers</Text>
+      </View>
+    </View>
+  );
+};
+
+export default FlightItem;
```
Comment (Astroyayo) [inline flight-reservation/components/FlightItemComponent/index.js:28]: Add an empty row at the end of the file

File: flight-reservation/components/FlightItemComponent/style.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        fontWeight: '600'
+    },
+    subText: {
+        fontSize: 12,
+        color: 'gray'
+    }
+
+});
+
+export default styles
```
Comment (Astroyayo) [inline flight-reservation/components/FlightItemComponent/style.js]: Add an empty row at the end of the file

**Review by Astroyayo — APPROVED** [review]
cool 👌

**Review by AmarilloVel — APPROVED** [review]
good job

**Pull request merged**